src/cron/hourlyNotifications.ts

```diff
@@ -0,0 +1,79 @@
+import { utcToZonedTime } from 'date-fns-tz';
+import {
+  schedulePersonalizedDigestSubscriptions,
+  notifyGeneratePersonalizedDigest,
+} from '../common';
+import {
+  User,
+  UserPersonalizedDigest,
+  UserPersonalizedDigestSendType,
+  UserPersonalizedDigestType,
+} from '../entity';
+import { Cron } from './cron';
+import { isWeekend, addHours, startOfHour, subDays } from 'date-fns';
+import { DEFAULT_TIMEZONE } from '../types';
+
+const sendType = UserPersonalizedDigestSendType.workdays;
+const digestTypes = [UserPersonalizedDigestType.StreakReminder];
```

Comment:
Will move the reading reminder into this in a different PR, as I want to deliver this by EOD